ostd/src/mm/page_table/mod.rs

```diff
@@ -380,4 +386,43 @@ pub trait PageTableEntryTrait:
     /// The level of the page table the entry resides is given since architectures
     /// like amd64 only uses a huge bit in intermediate levels.
     fn is_last(&self, level: PagingLevel) -> bool;
+
+    /// Converts the PTE into its corresponding `usize` value.
+    fn as_usize(self) -> usize {
+        // SAFETY: `Self` is Copy and has the same memory representation as usize
+        unsafe { transmute_unchecked(self) }
+    }
+
+    /// Converts a usize `pte_raw` into a PTE.
+    ///
+    /// # Safety
+    ///
+    /// The `pte_raw` must represents a valid PTE.
+    unsafe fn from_usize(pte_raw: usize) -> Self {
```

Comment:
`from_usize`/`as_usize` seems to be more suitable to be methods of `PodUsize`, right?